src/pe/section_table.rs

```diff
@@ -76,7 +76,9 @@ impl SectionTable {
         table.characteristics = bytes.gread_with(offset, scroll::LE)?;
 
         if let Some(idx) = table.name_offset()? {
-            table.real_name = Some(bytes.pread::<&str>(string_table_offset + idx)?.to_string());
+            if let Ok(real_name) = bytes.pread::<&str>(string_table_offset + idx) {
+                table.real_name = Some(real_name.to_string());
```

Comment:
i think it would be more idiomatic to do:

```rust
table.real_name = bytes.pread::<&str>(string_table_offset + idx).ok().map(String::to_owned);
```

the real question is whether:

1. It should be considered a fatal error if the name is not utf8
2. we should log on error, in which case keeping the `if let Ok` is better

Reply:
Sorry it took me so long to reply!
As far as I understand, `real_name` is stored in [COFF String Table](https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#coff-string-table), but there is nothing stated about the encoding. However? for [Symbol Name](https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#symbol-name-representation) the following is stated: 

> By convention, the names are treated as zero-terminated UTF-8 encoded strings.

Maybe, it can be applied to `real_name`, too? 

However, I do not consider it to be a fatal error, so maybe we should stick to `if let Ok`?